### PR TITLE
US 1583733 Add missing language IDs - 11

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for program output examples.
- Used "env" for Environment variable settings.
- Used "text" for example of base addresses.
- Used "xml" for Syntax examples.

Contributes to #2192

### CODE REVIEW

**Comment by tfosmark** [comment]
@v-thpra Please address the conflict. Thanks!

**Comment by v-thpra** [comment]
@mairaw Conflicts have been addressed and this PR should be ready to merge. Thanks!

**Comment by BillWagner** [comment]
We have another new merge conflict. Can you address it?

**Comment by v-thpra** [comment]
@BillWagner I addressed the conflict. This should be ready for review and merge. Thanks!

File: docs/framework/configure-apps/file-schema/runtime/compatsortnlsversion-element.md [hunk 1]
```diff
@@ -66,13 +66,13 @@ Specifies that the runtime should use legacy sort orders when performing string
   
  When you run the example on the .NET Framework 4, it displays the following output.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/runtime/compatsortnlsversion-element.md]:
```suggestion
 When you run the example on the .NET Framework 4, it displays the following output:
```

File: docs/framework/configure-apps/file-schema/runtime/compatsortnlsversion-element.md [hunk 2]
```diff
@@ -66,13 +66,13 @@ Specifies that the runtime should use legacy sort orders when performing string
   
  When you run the example on the .NET Framework 4, it displays the following output.  
   
-```  
+```output  
 sta follows a in the sort order.  
 ```  
   
  This is completely different from the output that is displayed when you run the example on the .NET Framework 3.5.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/runtime/compatsortnlsversion-element.md]:
```suggestion
 This is completely different from the output that is displayed when you run the example on the .NET Framework 3.5:
```

File: docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md [hunk 3]
```diff
@@ -8,13 +8,14 @@ Associates the factory used to produce the service host with the service to be h
   
 ## Syntax  
   
-```  
-<% @ServiceHost   
-Service = "Service, ServiceNamespace"   
+```xml  
+<% @ServiceHost
+Service = "Service, ServiceNamespace"
 Factory = "Factory, FactoryNamespace"
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md]:
```suggestion
Factory = "Factory, FactoryNamespace"
```

File: docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md [hunk 4]
```diff
@@ -8,13 +8,14 @@ Associates the factory used to produce the service host with the service to be h
...
 ## Syntax  
   
-```  
-<% @ServiceHost   
-Service = "Service, ServiceNamespace"   
+```xml  
+<% @ServiceHost
+Service = "Service, ServiceNamespace"
 Factory = "Factory, FactoryNamespace"  
 Debug = "Debug"
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md]:
```suggestion
Debug = "Debug"
```

File: docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md [hunk 5]
```diff
@@ -8,13 +8,14 @@ Associates the factory used to produce the service host with the service to be h
...
+```xml  
+<% @ServiceHost
+Service = "Service, ServiceNamespace"
 Factory = "Factory, FactoryNamespace"  
 Debug = "Debug"  
-Language = "Language"   
-CodeBehind = "CodeBehind"%>  
+Language = "Language"
+CodeBehind = "CodeBehind"
+%>
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md]:
```suggestion
%>
```

File: docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ Represents a configuration element that specifies a pass-through filter, which p
   
  For example, your site can contain the following base addresses.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md]:
```suggestion
 For example, your site can contain the following base addresses:
```

File: docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md [hunk 7]
```diff
@@ -51,7 +51,7 @@ Represents a collection of configuration elements that specify pass through filt
   
  For example, your site can contain the following base addresses.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md]:
```suggestion
 For example, your site can contain the following base addresses:
```

File: docs/framework/configure-apps/file-schema/wcf/commonparameters.md [hunk 8]
```diff
@@ -48,12 +48,13 @@ Represents a collection of parameters that are used globally across multiple ser
...
- `<add`  
-  
- `type="System.Workflow.Runtime.Tracking.SqlTrackingService, System.Workflow.Runtime, Version=3.0.00000.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"`  
-  
- `ConnectionString="Data Source=localhost;Initial Catalog=Partner20WFTP;Integrated Security=True;" />`  
+
+```xml  
+<add
+type="System.Workflow.Runtime.Tracking.SqlTrackingService, System.Workflow.Runtime, Version=3.0.00000.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" 
+
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/commonparameters.md]:
```suggestion
```

File: docs/framework/configure-apps/file-schema/wcf/commonparameters.md [hunk 9]
```diff
@@ -78,7 +79,7 @@ Represents a collection of parameters that are used globally across multiple ser
   
  The following sample code shows how to change the common parameters programmatically.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/commonparameters.md]:
```suggestion
 The following sample code shows how to change the common parameters programmatically:
```

File: docs/framework/configure-apps/file-schema/wcf/commonparameters.md [hunk 10]
```diff
@@ -78,7 +79,7 @@ Represents a collection of parameters that are used globally across multiple ser
   
  The following sample code shows how to change the common parameters programmatically.  
   
-```  
+```csharp  
 Configuration config=WebConfigurationManager.OpenWebConfiguration("/Workflow", "Default Web Site", null, "localhost");
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/commonparameters.md]:
```suggestion
Configuration config = WebConfigurationManager.OpenWebConfiguration("/Workflow", "Default Web Site", null, "localhost");
```

File: docs/framework/configure-apps/file-schema/wcf/commonparameters.md [hunk 11]
```diff
@@ -78,7 +79,7 @@ Represents a collection of parameters that are used globally across multiple ser
   
  The following sample code shows how to change the common parameters programmatically.  
   
-```  
+```csharp  
 Configuration config=WebConfigurationManager.OpenWebConfiguration("/Workflow", "Default Web Site", null, "localhost");  
 WorkflowRuntimeSection wfruntime=config.GetSection("WorkflowRuntime") as WorkflowRuntimeSection;
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/commonparameters.md]:
```suggestion
var wfruntime = config.GetSection("WorkflowRuntime") as WorkflowRuntimeSection;  
```

File: docs/framework/configure-apps/file-schema/wcf/commonparameters.md [hunk 12]
```diff
@@ -78,7 +79,7 @@ Represents a collection of parameters that are used globally across multiple ser
   
  The following sample code shows how to change the common parameters programmatically.  
   
-```  
+```csharp  
 Configuration config=WebConfigurationManager.OpenWebConfiguration("/Workflow", "Default Web Site", null, "localhost");  
 WorkflowRuntimeSection wfruntime=config.GetSection("WorkflowRuntime") as WorkflowRuntimeSection;  
 NameValueConfigurationCollection commonParameters=wfruntime.CommonParameters;
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/commonparameters.md]:
```suggestion
NameValueConfigurationCollection commonParameters = wfruntime.CommonParameters;
```

File: docs/framework/configure-apps/file-schema/windows-identity-foundation/issuertokenresolver.md [hunk 13]
```diff
@@ -64,9 +64,9 @@ Registers the issuer token resolver that is used by handlers in the token handle
 </issuerTokenResolver>  
 ```  
   
-## Example  
+## Example
   
-```  
+```csharp  
 public override void LoadCustomConfiguration(System.Xml.XmlNodeList nodelist)
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/windows-identity-foundation/issuertokenresolver.md]: The usage of `XmlNode` in the foreach loop indicates that there is already a `using System.Xml;` statement. Which means `System.Xml` is not needed here.

```suggestion
public override void LoadCustomConfiguration(XmlNodeList nodelist)
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Comment by Youssef1313** [comment]
My suggested changes are all unrelated to your changes.

**Comment by v-thpra** [comment]
@mairaw  I applied all of the suggested changes from @Youssef1313. Even though they are mostly unrelated to mine, they looked correct.

File: docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md [hunk 14]
```diff
@@ -47,9 +47,9 @@ Represents a configuration element that specifies a pass-through filter, which p
   
  IIS supports specifying multiple IIS bindings for each site, which results in multiple base addresses for each scheme. Because a WCF service hosted under a site allows binding to only one base address for each scheme, you can use the prefix filter feature to pick the required base address of the hosted service. The incoming base addresses, supplied by IIS, are filtered based on the optional prefix list filter.  
   
- For example, your site can contain the following base addresses.  
+ For example, your site can contain the following base addresses:
   
-```  
+```text
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md]: Let's keep this one without.

File: docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md [hunk 15]
```diff
@@ -49,9 +49,9 @@ Represents a collection of configuration elements that specify pass through filt
   
  IIS supports specifying multiple IIS bindings for each site, which results in multiple base addresses for each scheme. Because a WCF service hosted under a site allows binding to only one base address for each scheme, you can use the prefix filter feature to pick the required base address of the hosted service. The incoming base addresses, supplied by IIS, are filtered based on the optional prefix list filter.  
   
- For example, your site can contain the following base addresses.  
+ For example, your site can contain the following base addresses:
   
-```  
+```text
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md]: same here

**Review by mairaw — CHANGES_REQUESTED** [review]
Great changes @v-thpra. Left a few comments to be addressed.

**Comment by v-thpra** [comment]
@mairaw Thanks for the suggestions and comments. I implemented them all.

**Review by mairaw — APPROVED** [review]

**Pull request merged**